Configure

```diff
@@ -1193,7 +1193,9 @@ else               { $no_user_defines=1;    }
 
 # ALL MODIFICATIONS TO %config and %target MUST BE DONE FROM HERE ON
 
-unless ($disabled{engine}) {
+if ($disabled{engine}) {
+    $disabled{afalg} = "no-engine";
+} else {
```

Comment:
Would this auto-map to OPENSSL_NO_AFALG? Or put it differently, should it be "afalgeng"? (Because the define is OPENSSL_NO_AFALGENG).